### PR TITLE
Added transfer-call-inline-ncco.sh

### PR DESCRIPTION
**Pull request by Th3R4nd0m**
Hope this helps :3

### CODE REVIEW

File: voice/voice-api/transfer-call-inline-ncco.sh [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+source "../../config.sh"
+source "../../jwt.sh"
+
+curl -X POST https://api.nexmo.com/v1/calls/$UUID \
```
Comment (juliacodez) [inline voice/voice-api/transfer-call-inline-ncco.sh:5]:
```suggestion
curl -X PUT https://api.nexmo.com/v1/calls/$UUID \
```
This needs to be a PUT request.

File: voice/voice-api/transfer-call-inline-ncco.sh [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+  -H "Authorization: Bearer "$JWT \
+  -H "Content-Type: application/json"\
+  -d '{"action": "transfer",
+      "destination": {"type": "ncco", "ncco": ["action":"talk", "text":"This is a transfer action using an inline NCCO"]}}'
```
Comment (juliacodez) [inline voice/voice-api/transfer-call-inline-ncco.sh:9]:
```suggestion
      "destination": {"type": "ncco", "ncco": [{"action":"talk", "text":"This is a transfer action using an inline NCCO"}]}}'
```
The NCCO is an array of objects.

**Review by juliacodez — CHANGES_REQUESTED** [review]
Thank you for adding the curl code snippet for transferring a call with inline NCCO! 

This looks good, but there's a couple of changes that need to be made before we can merge: the transfer needs to be a PUT request, and the NCCO is an array of objects.

File: voice/voice-api/transfer-call-inline-ncco.sh (hunk 2 above)
Comment (Th3R4nd0m) [reply voice/voice-api/transfer-call-inline-ncco.sh]: Also fixed. My bad.

File: voice/voice-api/transfer-call-inline-ncco.sh (hunk 1 above)
Comment (Th3R4nd0m) [reply voice/voice-api/transfer-call-inline-ncco.sh]: Yes. My bad. Fixed.

**Review by juliacodez — APPROVED** [review]
Thank you, this is now ready to be merged. Happy Hacktoberfest!

**Pull request merged**